base/process.jl

```diff
@@ -2,22 +2,76 @@
 
 abstract AbstractCmd
 
+# libuv process option flags
+const UV_PROCESS_WINDOWS_VERBATIM_ARGUMENTS = UInt8(1 << 2)
+const UV_PROCESS_DETACHED = UInt8(1 << 3)
+const UV_PROCESS_WINDOWS_HIDE = UInt8(1 << 4)
+
 immutable Cmd <: AbstractCmd
     exec::Vector{ByteString}
     ignorestatus::Bool
-    detach::Bool
+    flags::UInt8 # libuv process flags
```

Comment:
Okay, I changed it.   I think it's a bit pointless to worry about ABI stability in this context, but 3 bytes are not worth arguing about.